client/src/config.js

```diff
@@ -5,4 +5,5 @@ export default {
   walletUrl: env.REACT_APP_WALLET_URL,
   gunUrl: env.REACT_APP_GUN_URL,
   apiUrl: env.REACT_APP_API_URL,
+  embedDataStudioUrl: env.REACT_APP_EMBED_DATA_STUDIO_URL,
```

Comment:
should be added to the env example (e.g. env.development in this package)
or better create separate .env.example containing all env vars available